src/components/HTMLEngineProvider/BaseHTMLEngineProvider.tsx

```diff
@@ -73,6 +73,11 @@ function BaseHTMLEngineProvider({textSelectable = false, children, enableExperim
                 mixedUAStyles: {...styles.mutedNormalTextLabel, ...styles.mb0},
                 contentModel: HTMLContentModel.block,
             }),
+            'centered-text': HTMLElementModel.fromCustomModel({
```

Comment:
Hmmm is it not possible to fix this with flex styles on the parent container(s)? Including styles to control layout in a translation file feels like an anti-pattern to me.

Reply:
Hmmm can you expand on that and provide some more details? If there's a problem with react-native-render-html, we have worked with the maintainer in the past and can try to sponsor a resolution to that problem

Reply:
@shubham1206agra brought details to slack [here](https://expensify.slack.com/archives/C07HPDRELLD/p1753333261643859?thread_ts=1753150848.710969&cid=C07HPDRELLD)